### DISCUSSION TITLE
Manual  install

### DISCUSSION THREAD

**Discussion by bottlesbo** [piitaya/lovelace-mushroom#1142] [Q&A]
Manual
Download mushroom.js file from the [latest-release].

Where is mushroom.js ?

**Comment by piitaya** [comment]
Hello. It's located here : https://github.com/piitaya/lovelace-mushroom/releases

![CleanShot 2023-05-11 at 09 59 48](https://github.com/piitaya/lovelace-mushroom/assets/5878303/ddf9e830-7309-4743-a8ab-950b918d907f)
